packages/twenty-front/src/modules/ai/graphql/queries/getAgentChatMessages.ts

```diff
@@ -7,7 +7,33 @@ export const GET_AGENT_CHAT_MESSAGES = gql`
       threadId
       role
       createdAt
-      rawContent
+      parts {
+        id
+        messageId
+        orderIndex
+        type
+        textContent
+        reasoningContent
+        toolName
+        toolCallId
+        toolInput
+        toolOutput
+        state
+        errorMessage
+        errorDetails
+        sourceUrlSourceId
+        sourceUrlUrl
+        sourceUrlTitle
+        sourceDocumentSourceId
```

Comment:
We don't manage sources right, do we? Could you please create an issue for it? Anything else you see missing?